docs/en/engines/table-engines/integrations/kafka.md

```diff
@@ -170,53 +170,37 @@ Similar to GraphiteMergeTree, the Kafka engine supports extended configuration u
   <kafka>
     <!-- Global configuration options for all tables of Kafka engine type -->
     <debug>cgrp</debug>
-    <auto_offset_reset>smallest</auto_offset_reset>
-	<statistics_interval_ms>600</statistics_interval_ms>
-
-    <!-- Configuration specific to topics "logs" and "stats" -->
-
-    <kafka_topic>
-      <name>logs</name>
-      <retry_backoff_ms>250</retry_backoff_ms>
-      <fetch_min_bytes>100000</fetch_min_bytes>
-    </kafka_topic>
-
-    <kafka_topic>
-      <name>stats</name>
-      <retry_backoff_ms>400</retry_backoff_ms>
-      <fetch_min_bytes>50000</fetch_min_bytes>
-    </kafka_topic>
+    <statistics_interval_ms>3000</statistics_interval_ms>
+
+    <!-- Settings for consumer -->
```

Comment:
Stupid question but is it no longer possible to have `<kafka_topic>` defined directly under `<kafka>`? Asking because the PR description says that is possible.

Reply:
Yes, it is possible to define `<kafka_topic>` directly under `<kafka>`. I'll add this case in the documentation.